src/variableSubstitution.ts

```diff
@@ -99,9 +99,12 @@ const VARIABLE_SUBSTITUTIONS: VariableSubstitution[] = [
 export function getVariableSubstitutedAssociations(associations: XMLFileAssociation[]): XMLFileAssociation[] {
 
   // Collect properties needed to resolve variables
-  const currentFile: TextDocument = window.activeTextEditor.document;
-  const currentFileUri: string = currentFile && currentFile.uri.fsPath;
-  const currentWorkspace: WorkspaceFolder = workspace.getWorkspaceFolder(currentFile && currentFile.uri);
+  const currentFile: TextDocument = (!!window.activeTextEditor && !!window.activeTextEditor.document && window.activeTextEditor.document.languageId === 'xml') ? window.activeTextEditor.document : undefined;
+  const currentFileUri: string = currentFile.uri && currentFile.uri.fsPath;
```

Comment:
I think you can write in one line:

const currentWorkspace: WorkspaceFolder = currentFile && currentFile.uri && workspace.getWorkspaceFolder(currentFile.uri);